includes/class-wc-gateway-ppec-cart-handler.php

```diff
@@ -458,17 +458,37 @@ public function enqueue_scripts() {
 			);
 
 		} elseif ( 'yes' === $settings->use_spb ) {
-			wp_register_script( 'paypal-checkout-js', 'https://www.paypalobjects.com/api/checkout.js', array(), null, true );
-			wp_register_script( 'wc-gateway-ppec-smart-payment-buttons', wc_gateway_ppec()->plugin_url . 'assets/js/wc-gateway-ppec-smart-payment-buttons.js', array( 'jquery', 'paypal-checkout-js' ), wc_gateway_ppec()->version, true );
+			// TODO: Add client ID.
```

Comment:
What's this?

Reply:
>  instead you provide a client ID when you load the new script and that is used to automatically detect your environment

This seems important? :)

Reply:
It is indeed! Currently a `TODO` since I don't know where (or even if) we store the client ID anywhere. @dechov might know?

I will clarify the list of TODO comments, and see if I'll have time soon to actually finish the implementation.

Reply:
Our backend integration uses the older NVP API, not the REST API, which is why we don't have a "PayPal REST client ID" to work with. If the newer SDK only supports the REST API, it would require a larger migration with new credentials required.

Reply:
> require a larger migration

You mean a lot more changes in this extension, right?

Reply:
Right, but also a migration plan spanning multiple releases, since a different set of credentials is required for the REST API.